omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -313,10 +292,10 @@ Basic setup for OMERO using NGINX::
     $ export HTTPPORT=${HTTPPORT:-8080}
     $ omero web config nginx-development --http $HTTPPORT > $(brew --prefix omero53)/etc/nginx.conf
 
-See installation script :download:`step03_nginx.sh <walkthrough/osx/step03_nginx.sh>`
-
-For detailed instructions on how to deploy OMERO.web in a production
-environment such as NGINX please see :doc:`install-web`.
+    $ mv /usr/local/etc/nginx/nginx.conf /usr/local/etc/nginx/nginx.conf.orig
+    $ bin/omero web config nginx-development > /usr/local/etc/nginx/nginx.conf
```

Comment:
Remove ``bin/`` (all the doc suggests to call ``omero ...``, probably an omission)